docs/docs/api-reference.md

```diff
@@ -15,6 +15,15 @@ This article discusses the API and props of **MuiChipsInput**. Props are defined
 <MuiChipsInput value={['foo', 'bar']} />
 ```
 
+## `addOnWhichKey`
+
+- Type: `string[]` | `string`
+- Default: `KEYBOARD_KEY.enter`
```

Comment:
explicit which key is it (Enter)

Reply:
There was an error in this PR, so I closed it and re-created it.
New PR: #50